src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -126,52 +129,43 @@ export default class AddStaffNotationDialog extends Vue {
   private notation = ''
 
   /** Court Order Number */
-  private courtOrderNumber = null as string
+  private courtOrderNumber = ''
 
   /** Whether filing has plan of arrangement. */
   private planOfArrangement = false
 
   /** Whether this component is currently saving. */
   private saving = false
 
-  /** Court Order component key */
+  /** Court Order component key, to force re-render. */
   private courtOrderKey = 0
 
-  /** Notation form validity */
+  /** Notation form validity. */
   private notationFormValid = false
 
-  /** Notation max length */
+  /** Notation max length. */
   private notationMaxLength = 2000
 
-  /** Flag to enable validation in this component */
+  /** Flag to enable validation in this component. */
   private enableValidation = false
 
-  /** Whether this filing is an administrative dissolution */
-  get administrativeDissolution (): boolean {
+  /** Whether this filing is an Administrative Dissolution. */
+  get isAdministrativeDissolution (): boolean {
     return this.isTypeAdministrativeDissolution({ name: this.name, dissolutionType: this.dissolutionType })
   }
 
-  /** Whether this filing is a put back on */
-  get putBackOn (): boolean {
+  /** Whether this filing is a Put Back On. */
+  get isPutBackOn (): boolean {
     return this.isTypePutBackOn({ name: this.name })
   }
 
+  /** The notation text area validation rules. */
   get notationRules (): Array<(v) => boolean | string> {
-    if (this.enableValidation) {
-      return [
-        (v: string) => !!v || ((this.administrativeDissolution || this.putBackOn) ? 'Enter a detailed comment'
-          : `Enter a ${this.displayName}`),
-        (v: string) => v.length <= this.notationMaxLength || 'Maximum characters exceeded.'
-      ]
-    } else return []
-  }
-
-  protected setFileNumber (courtOrderNumber: string): void {
-    this.courtOrderNumber = courtOrderNumber
-  }
-
-  protected setHasPlanOfArrangement (planOfArrangement: boolean):void {
-    this.planOfArrangement = planOfArrangement
+    return [
+      (v: string) => !!v || ((this.isAdministrativeDissolution || this.isPutBackOn) ? 'Enter a detailed comment'
+        : `Enter a ${this.displayName}`),
+      (v: string) => v.length <= this.notationMaxLength || 'Maximum characters exceeded.'
+    ]
```

Comment:
I moved the `enableValidation` check to the template. I think it's a bit cleaner this way and it saves a tiny bit of processing because this (cached) getter is only evaluated once (doesn't need to be responsive), while the component was already responsive to changed rule.

Ask if you have any questions.